chpl/chpl-resources/src/main/resources-production/jobs.xml

```diff
@@ -13,6 +13,26 @@
         <delete-triggers-in-group>ONC-ACBInheritedCertificationStatusErrorsReportTrigger</delete-triggers-in-group>
         <delete-triggers-in-group>ONC-ACBAllBrokenSurveillanceRulesReportTrigger</delete-triggers-in-group>
         <!-- END OCD-2944 -->
+        <delete-job>
```

Comment:
There's a PR out there that's removing these commands from this file. I don't know why this PR is adding them back in

Reply:
Those delete job elements are in the current jobs.xml that is on staging.  Didn't think I should be removing them.